index.js

```diff
@@ -12,31 +12,38 @@
  * @private
  */
 
+var EventEmitter = require('events').EventEmitter
+var fs           = require('fs')
+var path         = require('path')
+var Stream       = require('stream')
+var url          = require('url')
+
+var dirname   = path.dirname
+var join      = path.join
+var normalize = path.normalize
+var relative  = path.relative
+var resolve   = path.resolve
+
 var createError = require('http-errors')
-var debug = require('debug')('send')
+var destroy     = require('destroy')
+var escapeHtml  = require('escape-html')
+var etag        = require('etag')
+var fresh       = require('fresh')
+var mime        = require('mime')
+var ms          = require('ms');
+var onFinished  = require('on-finished')
+var parseRange  = require('range-parser')
+var statuses    = require('statuses')
+
+var debug     = require('debug')('send')
 var deprecate = require('depd')('send')
-var destroy = require('destroy')
```

Comment:
I just put the requires sorted alphabetically and grouped by build-ins, third-parties and local modules, as it's usually accepted in the Node.js community. Also I've made the requires style uniform, instead of some of them using its own `var` statement and others a common one, and some requires that were hidden between the variables declarations. Do you still want these clean-up changes to be undone?